community.md

```diff
@@ -1,11 +1,12 @@
 # Community And Ecosystem
 
-**KEY RECOMMENDATION**: Communities and ecosystems around open source
+**KEY RECOMMENDATION**: Note how an open source community generally follows a 
```

Comment:
Not just notice the fact, but identify the archetype(s) for the particular project in question?

Reply:
that was implied, but I'll revise to be explicit.